src/python/bot/fuzzers/builtin_fuzzers.py

```diff
@@ -35,3 +35,12 @@ def get(fuzzer_name):
     return None
 
   return BUILTIN_FUZZERS[fuzzer_name]
+
+
+def get_fuzzer_for_job(job_name):
+  """Return a fuzzer override for engine jobs."""
+  for fuzzer_name in BUILTIN_FUZZERS:
+    if fuzzer_name.lower() in job_name.lower():
```

Comment:
This problem was present before but I don't like that the way to determine a job uses an engine is if the engine is in the name. What if libFuzzer ends up in a project name?

Reply:
libFuzzer can't be in project name, a project can have multiple engines. we say to put engine name in job - https://google.github.io/clusterfuzz/setting-up-fuzzing/libfuzzer-and-afl/